dali/operators/reader/video_reader_op.h

```diff
@@ -65,30 +65,32 @@ class VideoReader : public DataReader<GPUBackend, SequenceWrapper> {
     DALI_ENFORCE(dtype_ == DALI_FLOAT || dtype_ == DALI_UINT8,
                  "Data type must be FLOAT or UINT8.");
 
-     enable_label_output_ = !file_root_.empty() || !file_list_.empty();
-     DALI_ENFORCE(enable_label_output_ || !enable_frame_num_,
-                  "frame numbers can be enabled only when "
-                  "`file_list` or `file_root` argument is passed");
-     DALI_ENFORCE(enable_label_output_ || !enable_timestamps_,
-                  "timestamps can be enabled only when "
-                  "`file_list` or `file_root` argument is passed");
+    enable_label_output_ = !file_root_.empty() || !file_list_.empty();
+    DALI_ENFORCE(enable_label_output_ || !enable_frame_num_,
+                "frame numbers can be enabled only when "
+                "`file_list` or `file_root` argument is passed");
+    DALI_ENFORCE(enable_label_output_ || !enable_timestamps_,
+                "timestamps can be enabled only when "
+                "`file_list` or `file_root` argument is passed");
 
     // TODO(spanev): Factor out the constructor body to make VideoReader compatible with lazy_init.
-      try {
-        loader_ = InitLoader<VideoLoader>(spec, filenames_);
-      } catch (std::exception &e) {
-        DALI_WARN(std::string(e.what()));
-        throw;
-      }
+    try {
+      loader_ = InitLoader<VideoLoader>(spec, filenames_);
+    } catch (dali::DALIException &e) {
+      throw e;
```

Comment:
This make sure we propagate the error to python unchanged.

Reply:
What is the difference with removing the try/catch? I am not sure I understand the logic behind this

Reply:
I assume we may get some exception not thrown by DALI,

Reply:
Removed